Yoast/wordpress-seo
# P1 193 elementor sidebar styling

## Context

* The sidebar as ported from the block editor to Elementor is missing some styles, mostly styles that are normally inherited from the block editor.

## Summary

This PR can be summarized in the following changelog entry:

* Fixes styling of Yoast SEO in the Elementor sidebar.

## Relevant technical choices:

* Decided to enqueue `admin-global.css` as it adds a lot of styles that were missing.

## Test instructions

This PR can be tested by following these steps:

* Merge
* Check whether the Yoast Sidebar in Elementor looks like it does in the Block editor. Please note Hover-color of text-links have been left in Elementor style deliberately for now, as adapting the gutenberg hover color looks strange.

## UI changes

## Documentation

## Quality assurance
